hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+</div>
+
+The following matching steps and tricks are proposed:
+
+- Calibrate your VNA for a frequency range larger than the intended bandwidth of the antenna.
+- Connect an RF coaxial cable to the RF line (for instance by soldering a pigtail to the line). Connect the RF coaxial cable to a VNA to measure the reflection coefficient, S11, looking into the antenna through the matching network.
+
+  a. Be sure to have a good connection to the ground plane to get the best electrical performance and the highest mechanical robustness during the measurement.
+
+  b. Make sure to route the pigtail towards the center of the PCB and then perpendicularly away from the PCB at the center point. This will limit the effect of the cable on the measured data as much as possible.
+
+- Start with no components on the antenna network shown in Figure 1.
+
+  a. The shunt components are not mounted.
+
+  b. The series component is not mounted.
+
+- Using port extension function in the VNA to move the reference point to the footprint of R1 and R2. This is achieved when the locus of the S-parameters in the Smith chart on the VNA have assembled in a point at the right edge of the Smith chart.
+
+- Mount a 0 Ω resistor at R2 in Figure 1.
+
+- Measure reflection coefficient of the center frequency of interest (the frequency half way between the lowest frequency and the highest frequency of the interesting band).
+
+- Using an online matching tool to calculate series and shunt component values to achieve 50 Ω match on the coaxial line. This will give a good starting point and a reasonable result at first attempt.
+
+- Iteratively change component values until the S11 is acceptable.
+  a. The standard matching criterion is either -6 dB or -10 dB reflection across all over the frequency band.
+  b. When this goal is achieved, it is recommended to use the same values on a small batch of boards to make sure that this matching is acceptable across production tolerances.
+
+
+# 3. Frequency error and HFXO Capacitor Bank(CTune) calibration 
+
+## 3.1. Why is Ctune calibration important?
+It is recommended to calibrate the HFXO frequency for the devices to ensure minimum error of the radio carrier frequency. The link quality will be broken or degenerated if the frequency offset(error) is big between two communication nodes. The CTune is in charge of tuning the RF carrier frequency because the HFXO clock is used for the radio carier frequency synthesization.
+All Silicon Labs modules and radio boards are factory calibrated, and the CTune value is stored on the device information page. For other Silicon Labs radio boards, it is equipped with an external EEPROM which stores the CTune value. 
+For custom boards, the frequency error or crystal calibration should be performed especially for narrow bandwidth communication. It is preferred to calibrate 10-20 board's HFXO frequency per design, and get the average CTune value which can be used for the design. 
```

Comment:
Should mention customers can choose to calibrate on a per-device basis in their own mfg test flow if desired.

Reply:
Added this suggestion.